planemo/github_util.py

```diff
@@ -55,12 +56,13 @@ def clone_fork_branch(ctx, target, path, **kwds):
             pass
```

Comment:
I think you should a `return` to `fork(ctx, path, **kwds)` 2 lines above.

Reply:
Yep, thanks!